app/backend/src/couchers/migrations/versions/3937eb39a4ca_add_host_request_archive_status.py

```diff
@@ -0,0 +1,26 @@
+"""Add host request archive status
+
+Revision ID: 3937eb39a4ca
+Revises: cd456767d2f7
```

Comment:
You'll just need to change this, and the other instance below to `18d100ed2803` instead once you rebase/merge develop into here.